routes/api/v1/favorites.js

```diff
@@ -14,9 +14,13 @@ router.get('/', (req, res) => {
         // retrieve an array of all user's favorite cities
         favoriteCities(user)
           .then(cities => {
-            console.log(cities)
+            // return forecast for each city's location in the array
+            forecastFor(cities)
+              .then(forecasts => {
```

Comment:
You might want to consider adding a catch here? 

Something like this? "          .catch(reason => response.send(reason.message))